build.gradle

```diff
@@ -6,20 +6,25 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.2.1"
-        classpath 'com.vanniktech:gradle-maven-publish-plugin:0.16.0'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
+plugins {
+  id 'signing'
+  id 'maven-publish'
+}
+
 allprojects {
     repositories {
         google()
         mavenCentral()
     }
+
 }
 
-task clean(type: Delete) {
+task cleanx(type: Delete) {
```

Comment:
?

Reply:
I got it. there's already a `clean` task from one of the plugins.

Reply:
had to disable that because a `clean` task already exists.